header.php

```diff
@@ -25,37 +25,34 @@
 <body <?php body_class( 'site-wrapper' ); ?>>
 	<a class="skip-link screen-reader-text" href="#main"><?php esc_html_e( 'Skip to content', '_s' ); ?></a>
 
-	<header class="site-header background-gallery">
-		<div class="display-flex container">
+	<header class="site-header bg-gray-200 py-12">
+		<div class="flex justify-between container my-0">
 
 			<div class="site-branding">
 
 				<?php the_custom_logo(); ?>
 
 				<?php if ( is_front_page() && is_home() ) : ?>
-					<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></h1>
+					<h1 class="site-title text-4xl"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" class="no-underline" rel="home"><?php bloginfo( 'name' ); ?></a></h1>
```

Comment:
Maybe this is a non-issue but if we ever have to update the Tailwind config, to change font sizes for example, unless we extend the default config or use the same names, these font sizes will be irrelevant.

Same goes for padding and colors. Especially colors. A mockup may not necessarily have "bg-gray-200".

Might be a non-issue because when building a new site, you'll have to come in and edit the header anyway and you can remove these at that time.

Reply:
Good call – we should make it as easy as possible to get things going without headaches or confusion. I think it'd be good to replace these types of text classes with the heading classes we usually use (like here, probably an `h3` class) and give some more thought to any spots we're using backgrounds or padding.